catkit2/services/oceanoptics_spectrometer_sim/oceanoptics_spectrometer_sim.py

```diff
@@ -0,0 +1,156 @@
+'''
+This module contains a service for simulation Ocean Optics Spectrometers.
+'''
+import numpy as np
+from catkit2.testbed.service import Service
+
+
+class OceanOpticsSpectrometerSim(Service):
+    '''
+    Service for Ocean Optics Spectrometer.
+
+    This service is a wrapper around the python seabreeze package.
+    It provides a simple interface to control the spectrometer and acquire spectra.
```

Comment:
This is not true for the simulated service and should be deleted.

Reply:
This still needs to go since it does not use seabreeze. I'd just delete both lines.